temporalcli/commandsmd/commands.md

```diff
@@ -2530,6 +2530,7 @@ Includes options set for [payload input](#options-set-for-payload-input).
 * `--yes`, `-y` (bool) -
   Don't prompt to confirm signaling.
   Only allowed when --query is present.
+* `--rps` (float) - Limit batch's requests per second. Only allowed if query is present.
```

Comment:
This will need to be rebased again since Andrew just merged an update that converted this file to YAML. :/ Hopefully will be straightforward, but let me know if it gets gnarly and/or you'd like me to do it.

Reply:
@josh-berry okay, rebased! Should be good now. 🤞